<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="LENOVO-PC" name="MavenPro_TYSS.SDET.Vtiger_MyCampaigns_TestScripts.TestCase_11" tests="2" failures="0" timestamp="2021-04-20T11:37:59 IST" time="0.001" errors="1">
  <testcase name="launchBrowser" time="0.001" classname="MavenPro_TYSS.SDET.Vtiger_MyCampaigns_TestScripts.TestCase_11">
    <error type="java.io.FileNotFoundException" message=".\SDET.Vtiger_MyCampaigns\data\commonData.property (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: .\SDET.Vtiger_MyCampaigns\data\commonData.property (The system cannot find the path specified)
at java.io.FileInputStream.open0(Native Method)
at java.io.FileInputStream.open(Unknown Source)
at java.io.FileInputStream.<init>(Unknown Source)
at java.io.FileInputStream.<init>(Unknown Source)
at com.Vtiger.MyCampaigns.generics.FileUtility.readDataFromPropFile(FileUtility.java:20)
at com.Vtiger.MyCampaigns.generics.Baseclass.launchBrowser(Baseclass.java:54)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- launchBrowser -->
  <system-out/>
  <testcase name="MyCampaigns_MassEdit" time="0.000" classname="MavenPro_TYSS.SDET.Vtiger_MyCampaigns_TestScripts.TestCase_11">
    <skipped/>
  </testcase> <!-- MyCampaigns_MassEdit -->
  <system-out/>
</testsuite> <!-- MavenPro_TYSS.SDET.Vtiger_MyCampaigns_TestScripts.TestCase_11 -->
